maintenance/importConfig.ts

```diff
@@ -245,6 +246,21 @@ async function cleanTmpDirectory(): Promise<void> {
 	console.log("temporary directory cleaned");
 }
 
+function matchId(
+	manufacturer: string,
+	prodType: string,
+	prodId: string,
+): boolean {
+	return (
+		program.ids !== undefined &&
+		program.ids.includes(
```

Comment:
program.ids?.includes

Reply:
isn't this ok as I checked `program.ids !== undefined` as first? If that fails the second condition is never checked

Reply:
yeah, but I prefer ?. in these cases.